app/eventPage/[id].tsx

```diff
@@ -72,17 +93,48 @@ export default function EventPage() {
           )}
         </YStack>
       </ScrollView>
-      {(typeof event.organizer !== "number" &&
-        user?.id === event.organizer.id) || (
-        <FloatingActionButton
-          iconAfter={
-            <IconSymbol name="square.and.pencil" size={24} color="$color8" />
-          }
-          // onPress={() => navigation.navigate("eventEdit", { eventId: event.docs[0].id })}
-        >
-          Edit Event
-        </FloatingActionButton>
-      )}
+      {joinStatus !== "joined" &&
+        typeof event.organizer !== "number" &&
+        user?.id === event.organizer.id && (
+          <FloatingActionButton
+            iconAfter={
+              <IconSymbol name="square.and.pencil" size={24} color="$color8" />
+            }
+          >
+            Edit Event
+          </FloatingActionButton>
+        )}
+      {joinStatus !== "joined" &&
+        typeof event.organizer !== "number" &&
+        user?.id !== event.organizer.id && (
+          <Link
+            push
+            href={`/joinEvent/attendeeDetails?code=${encodeURIComponent(event.eventCode)}&id=${encodeURIComponent(event.id)}`}
+          >
+            <FloatingActionButton
+              iconAfter={
+                <IconSymbol name="arrow.right" size={24} color="white" />
+              }
+              color="white"
```

Comment:
Haven't looked at how this turns out when running the app, but my hunch says that this should probably be `theme="accent"` instead 😅